packages/tasks/src/model-libraries.ts

```diff
@@ -933,6 +933,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path:"pytorch_model.bin" OR path:"model.safetensors"`,
 	},
+	torchgeo: {
+		prettyLabel: "TorchGeo",
+		repoName: "torchgeo",
```

Comment:
```suggestion
		prettyLabel: "TorchGeo",
		repoName: "TorchGeo",
```

Let's use the same pretty name (we'll soon merge those two fields to be the same)

Reply:
Done